### DISCUSSION TITLE
Calling `migrate --no-interaction` should create a Sqlite database, if it's missing

### DISCUSSION THREAD

**Discussion by danon** [laravel/framework#51861] [Ideas]
I'm trying to create a bash script that I can call to migrate a local sqlite database, or create it if it's missing.

Currently, when I run it, `php artisan migrate`, and the file doesn't exist, it asks me:
```
C:\> php artisan migrate

   WARN  The SQLite database configured for this application does not exist: database\database.sqlite.

  Would you like to create it? (yes/no) [yes]
❯ 
```

And that's a nice feature for a user interface, but that doesn't work well in a non-interactive mode such as in a bash script. When I use `--no-interaction` flag, then it just fails:
```

In SQLiteConnector.php line 34:
                                                                                                                                                                    
  Database file at path [C:\database\database.sqlite] does not exist. Ensure this is an absolute path to the database.  
                                                                                                                                                                    
```

I think it would be better if the file was just created. If it doesn't exist, then it's great.

I know I can use `--force`, but I don't want to put `--force` in my bash script, because if I do, and I mistakenly run it in production, it would wipe my production database, so I like the production warning. I don't want to use `--force`.

**Comment by ghost** [comment]
Hi, Can't you runt it with a -Y or --Y flag so it answers yes to all possible questions?

**Reply by danon** [reply]
There is no such option as `-Y` or `--Y`.

**Reply by danon** [reply]
```
C:\> php .\artisan migrate -Y

                                   
  The "-Y" option does not exist.  
                                   

C:\> php .\artisan migrate --Y   


  The "--Y" option does not exist.  


C:\> 
```
